cumulus_etl/etl/cli.py

```diff
@@ -267,11 +268,14 @@ async def etl_main(args: argparse.Namespace) -> None:
     job_context.last_successful_output_dir = args.dir_output
     job_context.save()
 
-    # If any task had a failure, flag that for the user
+    # Flag final status to user
+    common.print_header()
     failed = any(s.success < s.attempt for s in summaries)
     if failed:
-        print("** One or more tasks above did not 100% complete! **", file=sys.stderr)
+        print("🚨 One or more tasks above did not 100% complete! 🚨", file=sys.stderr)
```

Comment:
that's the stuff

Reply:
I literally can't believe I didn't start with those